integration-test/scripts/config-ums.sh

```diff
@@ -20,16 +20,14 @@ validate-ums-users() {
 
 prepare-cb-profile() {
     echo "Replacing UMS_HOST in profile file"
-    sed '/export UMS_HOST/d' ./integcb/Profile > ./integcb/Profile.tmp1
-    sed '/export UMS_PORT/d' ./integcb/Profile > ./integcb/Profile.tmp1
-    sed '/export CB_JAVA_OPTS/d' ./integcb/Profile.tmp1 > ./integcb/Profile.tmp
-    mv ./integcb/Profile.tmp ./integcb/Profile
+    sed -i '/export UMS_HOST/d' ./integcb/Profile
```

Comment:
Still not sure why I wrote this the way is with the multiple Profile.tmp files. Gues we'll find out.